### PR TITLE
Fixed the script error UI not shutting down the game properly

### LINKED ISSUE #8962: Lua exception dialog no longer works properly
The PR that removed the forced end-game screen regressed our handling of script errors.

The game will now throw up the regular game menu when the script crashes, with the error message showing as plain text below the center panel tabs (which are visually more prominent, so make the error less obvious).  The player can return to the game which then remains paused with no obvious indication why.

The return to game button should be disabled if the game was ended due to a script error, and the briefing tab should be hidden so that the error message takes priority.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/25409750-lua-exception-dialog-no-longer-works-properly?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F36085&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes #8962.

### CODE REVIEW

**Comment by penev92** [comment]
Looks good :+1: 
I'm not sure we want to disable `resumeButton` though. It might pay off to be able to return to looking at the map once you see there was a crash.

**Comment by Mailaender** [comment]
Enabled the `resumeButton` again.

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs [hunk 1]
```diff
@@ -78,7 +78,7 @@ public GameInfoLogic(Widget widget, World world, IngameInfoPanel activePanel)
 				numTabs++;
 				var debugTabButton = widget.Get<ButtonWidget>(string.Concat("BUTTON", numTabs.ToString()));
 				debugTabButton.Text = "Debug";
-				debugTabButton.IsVisible = () => lp != null && world.LobbyInfo.GlobalSettings.AllowCheats && numTabs > 1;
+				debugTabButton.IsVisible = () => lp != null && world.LobbyInfo.GlobalSettings.AllowCheats && numTabs > 1  && !hasError;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs]: Double space before the `&&`.

Comment (Mailaender) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoLogic.cs]: Woops. Fixed.

**Comment by abcdefg30** [comment]
:+1: / :white_check_mark:

**Pull request merged**